server/bleep/migrations/20230831165918_templates.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE templates (
+    id TEXT NOT NULL,
```

Comment:
Can we make this an `INTEGER PRIMARY KEY` instead? Sqlite will automatically give this column a default value if not specified, so we don't have to set the ID when inserting into the table. Integer keys are also easier to manage in general as they are `Copy`. In rust, this will be `i64`.